charts/lakefs/templates/NOTES.txt

```diff
@@ -0,0 +1,19 @@
+Thank you for installing lakeFS!
+
+1. Run the following To get a url to start setting up lakeFS:
+{{- if contains "NodePort" .Values.service.type }}
+  export NODE_PORT=$(kubectl get --namespace {{ .Release.Namespace }} -o jsonpath="{.spec.ports[0].nodePort}" services {{ template "lakefs.fullname" . }})
+  export NODE_IP=$(kubectl get nodes --namespace {{ .Release.Namespace }} -o jsonpath="{.items[0].status.addresses[0].address}")
```

Comment:
I think there are several IP types, like internal and external. It's probably better to choose the external one if exists.

Reply:
You're right, but it will be too much effort to decide which URL to take. In most places I found online they do it this way.
I think it's ok for now.